tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -470,7 +470,7 @@ TEXT_DRAW_TEST_F(CTLine, BasicDrawingTest, WhiteBackgroundTest<PixelByPixelImage
     CTRunDraw(run, context, {});
 }
 
-class Fonts : public WhiteBackgroundTest<PixelByPixelImageComparator<ComparisonMode::Mask>>,
+class Fonts : public WhiteBackgroundTest<PixelByPixelImageComparator<PixelComparisonModeMask<2300>>>,
```

Comment:
Yes. Given our reference images are 512x256, this is ~1.5%.  The bigger concern is that we are not accounting for cases such as lines being completely at a different offset, etc.  We need a more "fuzzy" proximity comparison than a pixel count.  But that is an exercise for a different time.

---
In reply to: [97449301](https://github.com/Microsoft/WinObjC/pull/1780#discussion_r97449301) [](ancestors = 97449301)